multiverse-source/src/point.rs

```diff
@@ -0,0 +1,38 @@
+use anyhow::Result;
+use dcspark_blockchain_source::{
+    cardano::{BlockEvent, CardanoNetworkEvent, Tip},
+    PullFrom,
+};
+
+pub trait Point: PullFrom + Sized + Clone + Send + Sync {
+    type V;
```

Comment:
the name of the type is not fully self-explanatory

Reply:
Same argument, I think it could be `MultiverseV`

Reply:
MultiverseV doesn't add much more context, if we replace it in #8 with GetFrom will be better